src/Test/L1/Worker/L1TestBase.cs

```diff
@@ -93,8 +93,7 @@ protected static TaskStep CreateScriptTask(string script)
             return step;
         }
 
-        protected async Task<TestResults> RunWorker(Pipelines.AgentJobRequestMessage message,
-            [CallerMemberName] string testName = "")
+        public void SetupL1([CallerMemberName] string testName = "")
```

Comment:
Added a separate `SetupL1` method, similar to what we have in the L0 tests. This allows us to set things up, adjust mocked values, then execute the test.